perform-release.sh

```diff
@@ -20,7 +20,7 @@ RELEASE_VERSION=$(getVersion)
 echo "Performing release ${RELEASE_VERSION}"
 
 echo "Deploying release ${RELEASE_VERSION}"
-./mvnw clean deploy -pl "\!osgi-support, \!osgi-support/mirror-subclass-plugin/, \!osgi-support/sdk-osgi-test-campaign/, \!osgi-support/commercetools-tests-fragment" -DskipTests -Prelease,javadoc-jdk-8u121
+./mvnw clean deploy -pl "!osgi-support, !osgi-support/mirror-subclass-plugin, !osgi-support/sdk-osgi-test-campaign, !osgi-support/commercetools-tests-fragment" -DskipTests -Prelease,javadoc-jdk-8u121
 
 echo "Publish release javadoc  ${RELEASE_VERSION}"
-./mvnw javadoc:aggregate scm-publish:publish-scm -Ppublish-site,javadoc-jdk-8u121
```

Comment:
@katmatt do we keep this line commented ?

Reply:
Good catch =) I commented that line so that we don't update our javadoc on gh pages while working on the automation. So please remove the comment.